### PR TITLE
Template sets google.android.annotations, opencensus,  perfmark (was grpc, now split in order to reduce size)

### PR DESCRIPTION
**Pull request by moljac**
### Describe your contribution

1. Moved `Xamarin.Io.OpenCensus.*` to this repo from: 
  https://github.com/xamarin/XamarinComponents/tree/main/Android/Io.OpenCensus
  - `io.opencensus:opencensus-api`
  - `io.opencensus:opencensus-contrib-grpc-metrics`
2. Moved `Xamarin.Io.PerfMark.PerfMarkApi` to this repo from:
  https://github.com/xamarin/XamarinComponents/tree/main/Android/Io.PerfMark
  - `io.perfmark:perfmark-api`
  
3. created template sets

## Updated/new (moved) artifacts

- com.squareup.okio:okio -  -> 1.13.0
- com.google.protobuf:protobuf-lite -  -> 3.0.1
- io.grpc:grpc-protobuf-lite -  -> 1.28.0
- io.grpc:grpc-stub -  -> 1.28.0
- io.grpc:grpc-okhttp -  -> 1.28.0
- io.grpc:grpc-context -  -> 1.28.0
- io.grpc:grpc-core -  -> 1.28.0
- io.grpc:grpc-android -  -> 1.28.0
- io.grpc:grpc-api -  -> 1.28.0
- io.opencensus:opencensus-api -  -> 0.28.3
- io.opencensus:opencensus-contrib-grpc-metrics -  -> 0.28.3
- io.perfmark:perfmark-api -  -> 0.23.0
- com.google.guava:guava -  -> 29.0-android
- com.google.android.datatransport:transport-runtime -  -> 3.0.1
- io.reactivex.rxjava2:rxandroid -  -> 2.1.1
- io.reactivex.rxjava2:rxjava -  -> 2.2.7
- com.squareup.picasso:picasso -  -> 2.71828
- com.google.code.gson:gson -  -> 2.8.6
- com.google.errorprone:error_prone_annotations -  -> 2.4.0
- com.google.android:annotations -  -> 4.1.1.4
- org.codehaus.mojo:animal-sniffer-annotations -  -> 1.18
- com.google.android.gms:play-services-ads-lite -  -> 20.0.0
- com.google.android.gms:play-services-ads-base -  -> 20.0.0
- com.google.android.gms:play-services-gass -  -> 20.0.0
- com.google.android.gms:play-services-measurement-base -  -> 16.3.0
- com.google.android.gms:play-services-measurement-api -  -> 16.3.0
- com.google.android.gms:play-services-measurement-base -  -> 17.0.1
- com.google.android.gms:play-services-measurement-api -  -> 17.0.1

### Does this change any of the generated binding API's?

`net6-android` TFM added

### Artifacts updated

Checklist
- [x] Add any metadata needed to get PR to compile
- [x] Verify any new namespaces are properly renamed (utilities.cake)
- [x] Verify nuget/namespace spelling (utilities.cake)

### CODE REVIEW

**Comment by moljac** [comment]
cleaned up PR 

I think I removed all that I could

File: config.json [hunk 1]
```diff
@@ -1623,6 +1623,58 @@
...
+        "templateSet": "opencensus"
+      },
+      {
+        "groupId": "io.perfmark",
+        "artifactId": "perfmark-api",
+        "version": "0.23.0",
+        "nugetVersion": "0.23.0",
+        "nugetId": "Xamarin.Io.PerfMark.PerfMarkApi",
+        "dependencyOnly": false,
+        "templateSet": "opencensus"
```
Comment (jpobst) [inline config.json:1668]: I think this should be "perfmark".

**Review by jpobst — COMMENTED** [review]
Thanks! Much easier to review!

I think there's one tiny issue and then this is ready.

File: config.json (hunk 1 above)
Comment (moljac) [reply config.json:1668]: wow
yes
good eye

Comment (moljac) [reply config.json]: fixed

**Review by jpobst — APPROVED** [review]

**Pull request merged**